openmdao/drivers/genetic_algorithm_driver.py

```diff
@@ -88,6 +97,8 @@ def _declare_options(self):
                              'as four times the number of bits.')
         self.options.declare('run_parallel', default=False,
                              desc='Set to True to execute the points in a generation in parallel.')
+        self.options.declare('procs_per_model', default=1,
+                             desc='Number of processors to give each model under MPI.')
```

Comment:
It might be more flexible to allow the user to specify how many cases they want to run in parallel, and then the proc allocator can decide if the resulting number of procs per model satisfies the model's min_procs/max_procs constraints.  This was the approach taken with doe driver.  (procs_per_model can effectively be set by passing min_procs in the add_subsystem call, so this might be a case of two ways of specifying the same thing..)

Reply:
What I did matches doe_driver except that I don't allow true/false.  Specifying number of cases to run in parallel rather than the number of procs your model needs seems like information that is further removed from what you know.  Guess we'd need some users to chime in.

Reply:
OpenMDAO does have the proc_allocator feature with the ability to specify min_procs/max_procs, which seems to directly target this kind of use case. If a model can use a varying number of procs, then let it work with what it's given.
My thought would be that the driver options should reflect the driver point of view (number of parallel cases) rather than the model perspective.

Reply:
As a user, I think the `procs_per_model` argument is a bit more natural. Lets go with that one. But there still needs to be an error is the proc_per_model/comm.size isn't an integer